frame/staking-rewards/src/lib.rs

```diff
@@ -181,7 +204,15 @@ pub mod pallet {
 		OnlyStakeOwnerCanUnstake,
 		/// Reward asset not found in reward pool.
 		RewardAssetNotFound,
+		// Review [Andy]
+		// missing documentation
```

Comment:
we use this error a few times throughout our codebase, it's basically for covering if the blocktime ever goes backwards (not that it ever should)